nilearn/image/image.py

```diff
@@ -1063,7 +1063,7 @@ def math_img(formula, **imgs):
     return new_img_like(niimg, result, niimg.affine)
 
 
-def binarize_img(img, threshold=0, mask_img=None):
+def binarize_img(img, threshold=0, two_sided=True, mask_img=None):
```

Comment:
Any reason not to add this new argument to the end? This will break code of people using positional arguments.

Reply:
Moved it in a new commit